src/state/apis/snapshot/snapshot.ts

```diff
@@ -79,9 +81,15 @@ export const snapshotApi = createApi({
             return bnOrZero(VotingPowerSchema.parse(votingPowerUnvalidated).vp)
           }),
         )
-        const data = BigNumber.sum(...votingPowerResults).toString()
-        console.log('addresses', evmAddresses, 'FOX voting power', data)
-        return { data }
+        const foxHeld = BigNumber.sum(...votingPowerResults).toNumber()
```

Comment:
yeah not sure if this will be a problem, us and snapshot api handling base units in vanilla `Number`s

Reply:
Yeah that's fine unless you hold 9,007,199,254,740,992 FOX - which is impossible since the current supply is 377,154,162 FOX, and the max/total supply is 1,000,001,337 FOX

Reply:
Note those are precision, not BNs

Reply:
ah easy as.